sympy/core/sympify.py

```diff
@@ -333,6 +346,11 @@ def sympify(a, locals=None, convert_xor=True, strict=False, rational=False,
         their SymPy equivalents. If True the expression will be evaluated
         and the result will be returned.
 
+    See Also
+    ========
+
+    parse_expr
```

Comment:
I think this needs to have the full path if it is not in the same file that it refers to: `sympy.parsing.sympy_parser.parse_expr`?

Reply:
I haven't really got my head round how it works but it doesn't seem like "see also" uses the same namespace as other sphinx directives.